### PR TITLE
Lead School to Lead Partner - Episode VIII

### PR DESCRIPTION
**Pull request by adamjcolvin**
### Context

[Trello](https://trello.com/c/lnNnUjlX)

This continues the refactoring of lead_school references throughout the app to lead_partner.

**Related PRs**
https://github.com/DFE-Digital/register-trainee-teachers/pull/4598 https://github.com/DFE-Digital/register-trainee-teachers/pull/4577 https://github.com/DFE-Digital/register-trainee-teachers/pull/4527 https://github.com/DFE-Digital/register-trainee-teachers/pull/4541 https://github.com/DFE-Digital/register-trainee-teachers/pull/4551 https://github.com/DFE-Digital/register-trainee-teachers/pull/4611 https://github.com/DFE-Digital/register-trainee-teachers/pull/4625

### Important business

- [ ] Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?
- [ ] Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?
- [ ] Do we need to send any updates to DQT as part of the work in this PR?
- [ ] Does this PR need an ADR?

NB: Please notify the #twd_data_insights team and ask for a review if new fields are being added to analytics.yml

### CODE REVIEW

File: spec/models/concerns/lead_school_migratable_spec.rb [hunk 1]
```diff
@@ -3,55 +3,55 @@
 require "rails_helper"
 
 describe LeadSchoolMigratable do
```
Comment (Reyko) [inline spec/models/concerns/lead_school_migratable_spec.rb:5]: If this is no longer needed, how about removing any remaining includes and delete the spec and the concern instead?

Comment (tomtrentham) [reply spec/models/concerns/lead_school_migratable_spec.rb:5]: Is this still being used in the `Funding::TraineeSummaryRow`?

**Review by tomtrentham — APPROVED** [review]
LGTM

File: spec/models/concerns/lead_school_migratable_spec.rb (hunk 1 above)
Comment (adamjcolvin) [reply spec/models/concerns/lead_school_migratable_spec.rb:5]: Yeah this is still being used for now. Once the TraineeSummaryRow has been refactored, then the whole thing can be removed.

**Pull request merged**